cmd/kube-apiserver/app/server.go

```diff
@@ -154,8 +154,6 @@ func CreateServerChain(runOptions *options.ServerRunOptions, stopCh <-chan struc
 		return nil, err
 	}
 
-	// TPRs are enabled and not yet beta, since this these are the successor, they fall under the same enablement rule
-	// If additional API servers are added, they should be gated.
```

Comment:
We should probably keep the line "If additional API servers are added, they should be gated."...

Reply:
ok